### ISSUE TITLE
compute router bgp keepalive_interval is not set

### ISSUE THREAD

**Issue by dlouvier** [terraform-google-modules/terraform-google-vault#171]
### TL;DR

As shown in the screenshot from the plan, keepalive_interval is not set and this value needs to be between 20 and 60. This causes every time plans runs and apply, the provider tries to replace this value causing necessary noise in the plan.

![image](https://user-images.githubusercontent.com/13359249/218711677-08e3ae5a-a96b-4726-a53f-b11a1e198645.png)

### Expected behavior

Set this value to it's default, 20, so plan doesn't show this again.

### Terraform Configuration

```hcl
n/a
```

### Terraform Version

```sh
❯ terraform version
Terraform v1.3.7
```

Other older versions also have the problem. This was introduced here: https://github.com/hashicorp/terraform-provider-google/blob/main/CHANGELOG.md#4130-march-7-2022 so you need to use google providers > v4.13
```


### Additional information

_No response_

**Comment by dlouvier** [comment]
The following PR fix the issue:
https://github.com/terraform-google-modules/terraform-google-vault/pull/169
